Model/IO/Export/Nastran/BulkData/BulkData.h

```diff
@@ -0,0 +1,40 @@
+//*************************
+//Created by Dominik Kopaczka
+//*************************
+
+#include<string>
+#include "../../../../Common/CommonMeshRepresentation.h"
+
+using std::string;
+
+#ifndef CPPWINLIN_BULKDATA_H
+#define CPPWINLIN_BULKDATA_H
+
+
+using std::string;
+namespace MeshIS
+{
+	namespace Model
+	{
+		namespace Export
+		{
+			namespace Nastran
+			{
+				namespace BData
+				{
+					class BulkData
+					{
+					public:
+						BulkData();
+						~BulkData();
+					public: string virtual get_GRID_line_from_vertex(MeshIS::Model::Common::Vertex vertex, int vertexID) = 0;
```

Comment:
What is a rationale to make this functions virtual?
And if they are made virtual, what other changes should be made to the class?